<!--REQUEST_ID=69172645-050f-4651-a5fe-5f9a61087077-->
Failure on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.46532&attempt_id=c9772462-4c75-4c9b-9025-4be63c55ba00)

Attempted: fstar, satysfi

<details><summary>Partial log (click to expand)</summary><p>

```

src/batBool.ml:111:  of_float (1. /. 0.) = false
///////////////////////////////////////////////////////////////////////////////
Ran: 1759 tests in: 36.71 seconds.
FAILURE
make: *** [Makefile:217: qtest-byte] Error 1
builder for '/nix/store/vc5kbzmin6m35gsfk44gw5if7i7scnx0-ocaml4.06.1-batteries-2.9.0.drv' failed with exit code 2
cannot build derivation '/nix/store/n95j4mr46fm6vb6zfsws4y67y1jixnr4-fstar-0.9.6.0.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/rr6aa2b68ydilfjmd4wq4jq9zdxfzhvr-satysfi-2018-03-07.drv': 1 dependencies couldn't be built
error: build of '/nix/store/n95j4mr46fm6vb6zfsws4y67y1jixnr4-fstar-0.9.6.0.drv', '/nix/store/rr6aa2b68ydilfjmd4wq4jq9zdxfzhvr-satysfi-2018-03-07.drv' failed
```
</p></details>

--- a/pkgs/development/ocaml-modules/batteries/default.nix
+++ b/pkgs/development/ocaml-modules/batteries/default.nix
@@ -13,7 +13,7 @@ stdenv.mkDerivation {
   buildInputs = [ ocaml findlib ocamlbuild qtest ];
   propagatedBuildInputs = [ num ];
 
-  doCheck = !stdenv.lib.versionAtLeast ocaml.version "4.07";
+  doCheck = !stdenv.lib.versionAtLeast ocaml.version "4.07" && !stdenv.isAarch64;
   checkTarget = "test test";
 
   createFindlibDestdir = true;

